OCA-UFCG/oca-frontend
# Refact/cache tiffs

This PR adds the caching feature to the map's URLs from GEE. Before, the request were made when the user changed the view and this were causing more slowness in the process of loading the maps. Now, we added a server-side caching script, which executes when the server is deployed and executes again to maintain the cache with the urls always updated.

## How to test it
1. use 'make run-dev' or 'make run-prod'
2. watch the POST requests logs and verify the time used to do the requests for the urls
3. the times should go over 60ms.

### screenshot
![image](https://github.com/user-attachments/assets/93578583-a225-4fe8-b3a0-fdf20299494f)
ps.: the first request can go over this time limit, because the cache is not initialized yet. to test without this issue,  you can hard refresh the page with "ctrl+f5" and see if the first POST request maintain the time low when requesting the urls.
